code/game/machinery/computer/shuttle_computers.dm

```diff
@@ -21,12 +66,21 @@
 	var/allow_silicons = 1		//If 0, AIs and cyborgs can't use this computer
 								//used for admin-only shuttles so that borgs cant hijack 'em
 
+	var/obj/item/weapon/disk/shuttle_coords/disk
+
 /obj/machinery/computer/shuttle_control/New()
 	if(shuttle)
 		name = "[shuttle.name] console"
 
 	.=..()
 
+/obj/machinery/computer/shuttle_control/Destroy()
+	..()
```

Comment:
Destroys should be done in reverse inheritance order i.e. put this below your new code.